Webtest.sh

```diff
@@ -41,6 +42,14 @@ if [ "$SIZE" -lt 50000 ]; then
   echo "Too small fetch for js: $SIZE"
   exit 1
 fi
+# Check if max payload set to value passed in cmd line parameter -maxpayloadsizekb
+SIZE=$($CURL "${BASE_URL}/echo?size=1048576" |wc -c)
+# It should return 8194, not sure why the difference
+if ! [ "$SIZE" -eq 8310 ]; then
```

Comment:
make it [ "$SIZE" -gt 8200 ] . so it's not brittle to 1 byte change in headers - or get the body only using -stdclient and then it should be exactly 8k